sdk/core/Azure.Core/samples/RequestContext.md

```diff
@@ -0,0 +1,45 @@
+# Azure client `RequestContext` samples
+
+**NOTE:** Samples in this file apply only to packages that follow [Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html). Names of such packages usually start with `Azure`.
+
+## Customize Error Handling
+
+You can suppress the exception a service method throws when an error response is returned.
+
+```C# Snippet:ErrorOptionsNoThrow
+RequestContext context = new RequestContext { ErrorOptions = ErrorOptions.NoThrow };
+```
+
+## Add Policy For Request
+
+You can add various types of policies for one request.
+
+### Add Policy Per Call
+
+```C# Snippet:AddPolicyPerCall
+var context = new RequestContext();
+context.AddPolicy(new AddHeaderPolicy("PerCallHeader", "Value"), HttpPipelinePosition.PerCall);
```

Comment:
I don't have the context for this. Could you help to explain the behaviour of adding this policy in the context?

Reply:
This will add a policy which will add a header to the request per call. Since it's a `RequestContext`, I assume this is useful in a LRO case in which there are multiple requests. And you can add the header to each request.

Reply:
I see. is it possible to put the descritpion above the code so reader could know what the below code would do.

Reply:
done